components/StyledComponents/StyledPopUpOverlay.js

```diff
@@ -0,0 +1,18 @@
+import styled from "styled-components";
+
+const PopUpOverlay = styled.div`
+  position: fixed;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  background-color: rgba(0, 0, 0, 0.5);
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  //the height of the Z index should ensure that this popup is always the top element.
+  //Since we work with the z-index in steps of 10, I have chosen 100 here.
```

Comment:
I guess I know why you put this comment.  But I think you can remove it before the merge.